internal/pkg/print/pretty/pretty.go

```diff
@@ -52,8 +52,7 @@ const (
 			{{- printf "\n" -}}
 			{{- range . }}
 				{{ printf "output.%s" .Name | colorize "\033[36m" }}
-				{{ tostring .Description | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}
-				{{ if $.Settings.OutputValues }}{{ .Value | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}{{ end}}
+				{{ tostring .Description | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}{{ if $.Settings.OutputValues }}{{- printf "\n" -}}{{ .Value | trimSuffix "\n" | default "n/a" | colorize "\033[90m" }}{{ end }}
```

Comment:
So ugly, but needed to keep the `pretty` output the same.

Reply:
Please take a look at https://github.com/segmentio/terraform-docs/pull/191#discussion_r376729470